src/classes/OPP_OpportunityNaming_BATCH_TEST.cls

```diff
@@ -1,10 +1,8 @@
 /*
-    Copyright (c) 2019, Salesforce.org
+    Copyright (c) 2015, Salesforce.org
```

Comment:
This class actually originated in 2019 so the copyright here and below should remain as 2019

Reply:
OK, I think I just reverted it to the prior one, but will change it.